boojum-cuda/native/ops_complex.cu

```diff
@@ -165,8 +165,17 @@ EXTERN __global__ void select_kernel(const unsigned *indexes, const base_field *
   memory::store_cs(dst + gid, value);
 }
 
-EXTERN __global__ void batch_inv_kernel(const base_field *src, base_field *dst, const unsigned count) {
-  constexpr unsigned INV_BATCH = 10;
+template <typename T> struct InvBatch {};
+template <> struct InvBatch<base_field> {
+  static constexpr unsigned INV_BATCH = 10;
+};
+template <> struct InvBatch<extension_field> {
+  static constexpr unsigned INV_BATCH = 6;
+};
+
+template<typename T, typename SETTER, typename GETTER>
+DEVICE_FORCEINLINE void batch_inv_impl(SETTER src, GETTER dst, const unsigned count) {
```

Comment:
Is the SETTER-GETTER meaning reversed here?

Reply:
oops, yes